datareduction/datareductionoperator.cpp

```diff
@@ -1862,7 +1862,7 @@ namespace DRO {
 
    bool VariableHeatFluxVector::reduceData(const SpatialCell* cell,char* buffer) {
       const Real HALF = 0.5;
-      # pragma omp parallel
+      #pragma omp parallel
```

Comment:
I guess this new heat flux also needs porting to ARCH?

Reply:
Done.